src/core/components/fields/LabeledTextAreaField.tsx

```diff
@@ -16,25 +16,29 @@ export interface LabeledTextAreaFieldProps
 
 export const LabeledTextAreaField = forwardRef<HTMLTextAreaElement, LabeledTextAreaFieldProps>(
   ({ name, label, outerProps, fieldProps, labelProps, ...props }, ref) => {
+    let validValue = (v) => (v === "" ? null : v)
+    let myType = props.type === "number" ? (Number as any) : validValue
```

Comment:
Could you provide some reasoning on this change? I do not see any issue with it but I also do not see the necessity of it – I can imagine there was an error/warning that you are correcting here, and it would be good to document it so we're on the same page.

Reply:
As mentioned in the commit message, this is done to fix the “transform error” which we are discussing on PR329. The reasoning can be found here:  https://github.com/STAPLE-verse/STAPLE/pull/329#discussion_r1794017022

I think a solution could be to create a different PR to refactor those lines because this code is found in several components.